index.html

```diff
@@ -54,26 +54,34 @@ <h1 class="intro_title">
             <fieldset class="help_wrap">
               <input
                 type="number"
-                id="fieldText"
-                placeholder="Укажите сумму в рублях"
+                class="fieldText"
+                placeholder="Укажите сумму в рублях."
+                required
               />
-              <button id="btСalculation">Помощь с выбором</button>
-              <p class="result">
+              <button class="btCalculation">Помощь с выбором</button>
+              <p class="help">
                 Результат:
-                <output type="text" id="fieldResult"></output>
+                <output type="text" class="fieldResult"></output>
```

Comment:
я может путаю, а так нельзя сделать?

```suggestion
                <output type="text" class="fieldResult" />
```